panther_driver/src/panther_can.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/python3
```

Comment:
Preferably there should be lock on top of the class and not within it. Meaning access to anything within the class should be guarded by lock outside of the scope of this class